### ISSUE TITLE
Error thrown when headers below level are present

### ISSUE THREAD

**Issue by alexkwolfe** [valeriangalliat/markdown-it-anchor#7]
When rendering a document with headers that are below the specified `level`, an error is thrown. The below example illustrates the problem. 

If the `# Title` declaration is removed, the error is no longer thrown. So it appears that headers below the specified level cause this error. 

```coffeescript
markdownIt = require('markdown-it')

content = """
# Title

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.

## Sub Title

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.

### Sub Sub Title

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
"""

rendered = markdownIt()
  .use require('markdown-it-anchor'), level: 2, permalink: true
  .render(content)

console.log rendered
```

**Comment by valeriangalliat** [comment]
Hi, thank you for reporting this, and sorry for the late reply!

This is a bug due to the permalink renderer being called for all headers regardless they match the level requirements.

Fixed with 2.2.1.

**Comment by alexkwolfe** [comment]
No need to apologize! Thanks so much for all the work you've done on this module. :+1:
